src/Plugin/Select/Handler.php

```diff
@@ -443,10 +462,9 @@ protected static function handleSelectCountOnField(Client $manticoreClient, Payl
 	 * Remove table alias syntax from query if exists
 	 *
 	 * @param Payload $payload
-	 * @param string $query
 	 * @return void
 	 */
-	protected static function checkQueryForAliasSyntax(Payload &$payload, string &$query): void {
+	protected static function checkQueryForAliasSyntax(Payload &$payload): void {
```

Comment:
Objects are passed by reference by default. We should remove the '&' as it serves no purpose here.